src/net.h

```diff
@@ -359,15 +360,13 @@ class CNode
 
 private:
 
-	CNode(const CNode&);
+    CNode(const CNode&);
     void operator=(const CNode&);
 
-	// Network usage totals
-	 static CCriticalSection cs_totalBytesRecv;
-     static CCriticalSection cs_totalBytesSent;
-     static uint64_t nTotalBytesRecv;
-     static uint64_t nTotalBytesSent;
- 
+    // Network usage totals
+    static std::atomic<uint64_t> nTotalBytesRecv;
+    static std::atomic<uint64_t> nTotalBytesSent;
```

Comment:
Started looking into this and I'm not sure we should change that. At least not just that. The variables are members of the `CNode` class and are accessed via the same class. By moving them to the cpp they are no longers a member of the class, but they are still accessed through it. If we move them to the cpp we should also move the accessor functions out of the class. There will still only be one instance of them as they're a part of the class.

In Bitcoin these variables have been moved into a `CConnman` class without being static.